python/CHANGELOG.rst

```diff
@@ -1,3 +1,13 @@
+--------------------
+[0.5.1] - 2022-XX-XX
+--------------------
+
+**Changes**
+
+- Add ``Tree.edge_array`` and ``Tree.edge``. Returns the edge id of the encoding
+  the relationship of each node with its parent.
+  (:user:`GertjanBisschop`)
```

Comment:
Is there an issue and PR num we can put here?

Reply:
Good point. Will make an issue and add a PR num. And add the changes to the 0.5.0 release section.